en_US/changes/changes-4.3.md

```diff
@@ -15,6 +15,72 @@ ref: undefined
 
 # Changes
 
+## Version 4.3.9
+
+*Release Date: 2021-11-02*
+
+EMQ X 4.3.9 is released now, it mainly includes the following changes:
+
+**Bug fixes:**
+
+- Fix the issue that WebHook's TLS is unavailable
+
+  Github PR: [emqx#5696](https://github.com/emqx/emqx/pull/5696)
+
+- Fix the issue that MongoDB resources do not support domain names
+
+  Github PR: [emqx#6035](https://github.com/emqx/emqx/pull/6035)
+
+- Fix the performance of ACL based on the built-in database
+
+  Github PR: [emqx#5885](https://github.com/emqx/emqx/pull/5885)
+
+- Fix the issue that the authentication based on the built-in database incorrectly transcodes the HTTP request parameters
+
+  Github PR: [emqx#5674](https://github.com/emqx/emqx/pull/5674)
+
+- Fixed the issue that the Client ID containing "\" characters could not be searched in a fuzzy manner
```

Comment:
```suggestion
- Allow "\" characters in client ID (which previously affected fuzzy lookup)
```

Reply:
The "\" character was previously supported, but it will affect fuzzy queries. The new description looks like we did not support client connections that contained "\" characters.